xml/System.Reflection.Metadata.Ecma335/SignatureDecoder`2.xml

```diff
@@ -186,10 +195,11 @@
         <Parameter Name="allowTypeSpecifications" Type="System.Boolean" />
       </Parameters>
       <Docs>
-        <param name="blobReader">To be added.</param>
-        <param name="allowTypeSpecifications">To be added.</param>
-        <summary>To be added.</summary>
-        <returns>To be added.</returns>
+        <param name="blobReader">The blob reader positioned at the leading SignatureTypeCode</param>
```

Comment:
Is there a cref for SignatureTypeCode. Or a link we can add to the remarks section?

Reply:
Yes (found [here](https://github.com/dotnet/dotnet-api-docs/blob/master/xml/System.Reflection.Metadata/SignatureTypeCode.xml)):
```suggestion
        <param name="blobReader">The blob reader positioned at the leading <see cref="T:System.Reflection.Metadata.SignatureTypeCode" />.</param>
```